docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+    However, it is a bit too easy to fall into the assumption that Wowhead data is correct and complete when it really isn't. Wowhead's data comes from user submissions, and has its own issues with things like drop rates as a result. For all its apparent authority, WH data can be fragmentary, incomplete and misleading if not interpreted with care.
+	Note the two versions listed here. TBC Wowhead is usually more helpful, but the current live version of Wowhead does also contain user comments from the Wrath era, which TBC WH doesn't.  
+
+- [Wowpedia](https://wowpedia.fandom.com/wiki/Special:Search) - Wowpedia can be a pretty good source, as it has a history function that lets you view an article as it was during the WotLK era. To access it, go to the relevant article, and look to the right of the article title. Next to 'View Source' you will see 3 vertically arranged dots. Clicking that allows access to the History of the article. You typically want the last article written before the date of 12 October 2010 (Cata's release date).
+
+    Wowpedia often aggregates patch notes relating to a particular talent or ability, which is frequently useful. It also often records quest text and quest-giver dialogue, so can be a handy source for that sort of thing.
+
+- YouTube - can be an excellent source of information, as there are lots of videos of people doing quests in Classic/TBC. Sadly there are few from the original WotLK era. The other problem with YT is that retail players are not the only ones putting Wow videos up. Players on other private servers also make videos (watch out for one in particular called Bue, who will often be top result when searching for a quest on YouTube.) These private server videos are of course largely useless as sources for correct Blizz-like behaviour.
+
+- [TrinityCore](https://github.com/TrinityCore/TrinityCore/issues) - Trinity is older and larger than AC, and their own bug tracker and issues database is correspondingly larger than ours. As a result it is often a good place to check for other reports of an issue if the AC tracker doesn't have any results. You can also quite often find a solution for the issue as well. Be aware that TC has multiple branches, only one of which is targeting 3.3.5a.
+
+- Sniffs data - the new hotness. 'Sniffs' are intercepted data packets players have captured when playing retail Wow. They have the advantage of using real retail data and making it available in SQL, and thus broadly useable. The disadvantages are that the sniff data only contains client-visible data, it is often incomplete and patchy, and is often difficult to otherwise verify. 
```

Comment:
New, no. The best we can get to blizzlike, yes :D

Reply:
Removed that bit, and bumped the sniffs listing up a couple of rungs to emphasise its importance.